src/commands/createFunctionApp/FunctionAppRuntimeStep.ts

```diff
@@ -48,10 +54,12 @@ export class FunctionAppRuntimeStep extends AzureWizardPromptStep<IFunctionAppWi
                 picks.unshift({ label: 'Node.js 12.x', data: 'node|12' });
             }
 
-            if (context.newSiteOS === WebsiteOS.linux) {
+            if (context.newSiteOS !== WebsiteOS.windows) {
```

Comment:
How would you get here with an OS already selected? Is there a programmatic setting?

Reply:
Basic create - for example, we default to linux for python projects, but still prompt them to see if they want 3.7.x or 3.6.x

Might be other cases, but that's the main one.